src/item.cpp

```diff
@@ -10112,7 +10112,7 @@ units::volume item::check_for_free_space( const item *it ) const
                 }
             }
         } else {
-            if( container->contents.contents_are_rigid() ) {
+            if( container->contents.all_pockets_rigid() ) {
```

Comment:
So we might want to check there are pockets here, assuming the original logic is correct.

Reply:
Oh, I somehow missed that. However the else case this is called in already requires `item_contents::get_all_contained_pockets` to not find any pockets, so if that logic is correct, the entire else case wouldn't be needed.

Reply:
Checked the method and where it's used to make sure. The else case really is unnecessary since all the pockets are already handled in the if case, so I removed it completely.